src/DirectusAsset.tsx

```diff
@@ -1,11 +1,26 @@
+import * as React from 'react';
+
 import { DirectusAssetProps } from './types';
+import { DirectusContext } from './DirectusProvider';
 
-export const DirectusAsset = ({ apiUrl, asset, download = false, render }: DirectusAssetProps): JSX.Element => {
+export const DirectusAsset = ({
+  apiUrl: propsApiUrl,
+  asset,
+  download = false,
+  render,
+}: DirectusAssetProps): JSX.Element => {
+  const directusContext = React.useContext(DirectusContext);
   let params = {};
   if (download) {
     params = { ...params, download: '' };
   }
 
+  if (!directusContext && !propsApiUrl) {
```

Comment:
Maybe move this check before let params?

Reply:
Yes makes sense, I get right on it